timely_beliefs/beliefs/classes.py

```diff
@@ -52,18 +54,31 @@ def __init__(
         self,
         sensor: Sensor,
         source: Union[BeliefSource, str, int],
-        value: float,
+        event_value: Optional[float] = None,
```

Comment:
I'd add a comment here that `value` and `event_value` are the same with just different names. Really, I think I'd like to keep one parameter here. If you want to move towards `event_value`, we should do that whole-hearted. Maybe deprecate `value`?

Reply:
This is an attempt to migrate away from inconsistent variable naming. But instead of immediately replacing the variable name, I wanted to try out writing a user warning for an upcoming deprecation. I'd like to deprecate it in a later (major) release.